composer.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "multi-safepay/presta-shop",
```

For a friendly composer require. I want to remove the dash (-)
```suggestion
  "name": "multisafepay/prestashop",
```